misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,44 @@
+import datetime
```

Comment:
Please be explicit when importing the datetime module, and replace this line with `from datetime import timedelta`.